README.md

```
@@ -51,73 +64,61 @@ npm run build
+         - "3000:3000"
+       environment:
+         - VERCEL_ACCESS_TOKEN=your_token_here
+         - NODE_ENV=production
+       restart: unless-stopped
    ```
 
-   For Windows:
+#### 🖥️ IDE Integration
+
+**For Cursor IDE or Codeium Cascade:**
+- Server URL: `http://localhost:3000/mcp`
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)